doc/api/fs.md

```diff
@@ -1381,6 +1381,15 @@ the specified file descriptor. This means that no `'open'` event will be
 emitted. Note that `fd` should be blocking; non-blocking `fd`s should be passed
 to [`net.Socket`][].
 
+Also note that the blocking fd, if pointing to a character device (such as
+keyboard or sound card) can potentially block the main thread on stream close.
+This is because these devices do not produce EOF character as part of their data
+flow cycle, and thereby epitomize endless streams. As a result, they do not
```

Comment:
I don't think "epitomize" is a word most non-english speakers will know.

Reply:
please suggest alternative, if you know.

Reply:
just endless streams straight up? i don't personally know what epitomize mean

Reply:
changed the wording from epitomize to exemplifies